Packs/MicrosoftExchangeOnPremise/ReleaseNotes/2_1_2.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### EWS v2
+
+Fixed an issue so that EML files downloaded from ***ews-get-items-as-eml*** command and **parse_incident_from_item** will now retain CRLF line separators.
```

Comment:
```suggestion
Fixed an issue so that EML files downloaded from the ***ews-get-items-as-eml*** command and **parse_incident_from_item** will now retain CRLF line separators.
```